components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+
+### Omega will use TEOS10 for the equation of state.
+See additional [EOS design document](EOS)
+
+### Omega-1.0 will add new terms for the pressure gradient, vertical mixing, and vertical advection.
+See forthcoming design documents on the pressure gradient, vertical mixing, and vertical advection.
+
+## 3. Conservation Equations
+
+### Control Volume Formulation
+
+We begin with the continuous, control-volume form of the conservation equations. Consider an arbitrary control volume $V(t)$ with a bounding control surface $\partial V(t)$. The fluid has density $\rho({\bf x},t)$ and velocity ${\bf v}({\bf x},t)$. The control surface is moving at a velocity ${\bf v}_r({\bf x},t)$. The conservation equations are
+
+mass:
+
+$$
+\frac{d}{dt} \int_{V(t)} \rho({\bf x},t) \, dV
++ \int_{\partial V(t)}\rho({\bf x},t)\left({\bf v}({\bf x},t) - {\bf v}_r \right) \cdot {\bf n} \, dA
+= 0
+$$ (continuous-mass)
+
+tracers:
+
+$$
+\frac{d}{dt} \int_{V(t)} \rho({\bf x},t) \, \varphi({\bf x},t) \, dV
++ \int_{\partial V(t)}\rho({\bf x},t)\, \varphi({\bf x},t) \left({\bf v}({\bf x},t) - {\bf v}_r \right) \cdot {\bf n} \, dA
+= 0
+$$ (continuous-tracer)
+
+momentum:
+
+$$
+&\frac{d}{dt} \int_{V(t)} \rho({\bf x},t)\,  {\bf v}({\bf x},t) \, dV
++ \int_{\partial V(t)}\rho({\bf x},t)\, {\bf v}({\bf x},t) \left({\bf v}({\bf x},t) - {\bf v}_r \right) \cdot {\bf n} \, dA
+\\ & \; \; \; =
+\int_{V(t)} \rho({\bf x},t) \, {\bf g}\, dV
++ \int_{\partial V(t)} {\bf f}({\bf n},{\bf x},t)  \, dA
+$$ (continuous-momentum)
+
+These equations are taken from [Kundu et al. 2016](https://www.amazon.com/dp/012405935X/) (4.5) for mass conservation and (4.17) for momentum conservation.
+All notation is identical to Kundu, except that we use ${\bf v}$ for the three-dimensional velocity (${\bf u}$ will be used below for horizontal velocities) and ${\bf v}_r$ and $\partial V$ match the notation in [Ringler et al. 2013](https://www.sciencedirect.com/science/article/pii/S1463500313000760) Appendix A.2.
+
+
+The tracer equation is simply mass conservation, where the conserved quantity is the tracer mass $\rho \varphi$, as $\varphi$ is the tracer concentration per unit mass.
+In all three equations, the first term is the change of the quantity within the control volume; the second term is the flux through the moving boundary.
+If the control surface moves with the fluid in a Lagrangian fashion, then ${\bf v}_r={\bf v}$ and the second term is zero.
+
+The momentum equation  is an expression of Newton's second law and has two additional terms on the right hand side.
+The first additional term is the body force, $\rho {\bf g} dV$, where ${\bf g}$ may be expressed as the gradient of a potential ${\bf g}= - \nabla_{3D} \Phi$ for conservative body forces.
+Note ${\bf g}$ is general here and not yet Earth's gravity.
+Body forces also include forces from a rotating frame of reference, such as the Coriolis force.
```

Comment:
Should we include a general body force term that is not gravity then?

Reply:
Yes, I think that makes more sense (and speaks for not calling it `g`, then).